crates/fluvio-spu/src/services/auth/root.rs

```diff
@@ -0,0 +1,56 @@
+use async_trait::async_trait;
+use fluvio_auth::{AuthContext, AuthError, Authorization, InstanceAction, TypeAction};
+use fluvio_controlplane_metadata::extended::ObjectType;
+use fluvio_socket::FluvioSocket;
+use tracing::instrument;
+
+/// Authorization that allows by remote Id
+#[derive(Debug, Clone)]
+pub struct SpuRootAuthorization {}
+
+#[async_trait]
+impl Authorization for SpuRootAuthorization {
```

Comment:
Why do you need SPU Authorization since it allows for everything?  Let's not try to re-map same semantics as SC which may not be same meaning

Reply:
Because I am calling an `AuthContext` method, the `allow_remote_id`.

Should I rename this default authorization?

Reply:
I don't undestand. 

should I not check authorization like it:

```rust
// authorization check
if !auth_ctx
    .auth
    .allow_remote_id(&req_msg.request.remote_cluster_id)
{
    warn!(
        "identity mismatch for remote_id: {}",
        req_msg.request.remote_cluster_id
    );
    return;
}
```

Reply:
Something like this:
```
if let Ok(authorized) = auth_ctx
        .auth
        .allow_instance_action(RemoteSpec::Edge, InstanceAction::Cluster, cluster_id)
        .await
    {
```

Reply:
Auth context is always there.  It will be configured differently on whether we configure with TLS or not.
If no TLS then it will be set to Root level authorization ( means you can do whatever you want).  If TLS then authorization is derived from TLS certs